README.md

```diff
@@ -1,10 +1,55 @@
 # MuckRock
+**MuckRock** &middot; [Squarelet][squarelet] &middot; [DocumentCloud][documentcloud] &middot; [DocumentCloud-Frontend][documentcloudfrontend]
 
 [![Codeship Status for MuckRock/muckrock][codeship-img]][codeship]
 [![codecov.io][codecov-img]][codecov]
 
 MuckRock is a non-profit collaborative news site that gives you the tools to keep our government transparent and accountable.
 
+## Prerequisites 
+MuckRock depends on Squarelet for user authentication. As the services need to communivate directly, the development environment for MuckRock depends on the development environment for Squarelet - the MuckRock docker containers will join Squarelet's docker network. [Please install Squarelet and set up its development environment first][squarelet].
+
+### Squarelet Integration
+The Squarelet project provides an authentication system for MuckRock. Therefore, there must be certain things set up within Squarelet before you can begin accessing them individually. 
+
+ 1. With the Squarelet containers running, in your terminal within the Squarelet folder, open up the bash shell with `inv sh`
+ 2. Within the bash shell, utilize this command to create your RSA key `./manage.py creatersakey`
+ 3. Then create a superuser utilizing the following command: `./manage.py createsuperuser`. 
+ 4. Exit the bash shell with the command `exit`
+ 5. Enter the Django shell with the command `inv shell`
+ 6. Grab the user you have just created with the following command
+```python
+tempUser = User.objects.all()[0]
+```  
+*If you created multiple users you will have to use Django queries to grab the exact user you want.*
+
+ 7. Then set the `tempUser` variable's `is_staff` value equal to true, and save the user. 
+ ```python
+tempUser.is_staff = True
+```  
+8. Exit the Django shell with the command `exit`
+9. In a browser navigate to the admin portal. You can find this portal by using the base Squarelet URL, and appending /admin to the end of it.
+10. Using the credentials you just created, login. *If you find yourself being redirected to the login page on successful credentials, change the browser you are using.* 
+11. Navigate to [Clients](https://dev.squarelet.com/admin/oidc_provider/client/)
+12. Create a client called `MuckRock Dev`
+13. Make sure the fields have the following values:
+
+|Field| Value |
+|--|--|
+| Owner | blank/your user account |
+|Client Type|Confidential|
+|Response Types|code (Authorization Code Flow)|
+|Redirect URIs (on separate lines)|http://dev.muckrock.com/accounts/complete/squarelet http://dev.foiamachine.org/accounts/complete/squarelet|
```

Comment:
All links should be HTTPS

Did you get it working with HTTPS?

Reply:
the links were copied from the existing documentation, but thanks for the catch - updating them all to https :) 

I was personally unable to get muckrock/muckrock running because the node-sass package that the frontend is using does not compile on an M1 chip *last time i checked.

"Error: Node Sass does not yet support your current environment: OS X Unsupported architecture (arm64) with Unsupported runtime (93)"